### DISCUSSION TITLE
Can we include content information about product labeling as it pertains to it's lifecycle/state in Paste?

### DISCUSSION THREAD

**Discussion by mjanssen2021** [twilio-labs/paste#2181] [Ideas] [answered]
My team and I in account security have several product versions that are in the EOL process currently, and I asked around for what other designers use to denote their products as being in different life cycle stages. This is the list I have currently - could we include this in the paste content guidelines (pending caroline teks or a content authors review)? I do not believe this is the complete list as we have other states like private beta, etc... 

**Including guidance that these terms need to be in both the page title and navigation**

- **Product name (Beta)**: This is a product recently launched in beta mode. Both the navigation and the title should have the twilio "beta" tag on them with the ability to click on the beta tag to popup the beta description modal. This tag should be taken off when a product is GA. 
-  **Product name only**: This is a product actively being used by customers and is the most up to date version.
- **Product name (Classic)**: This is a product actively being used by customers, but is not the most up to date version. It is a version customers like to use, but there are no plans to deprecate it at this time. 
- **Product name (Legacy)**: This is a product actively being used by customers, it is not the most recent version, and we want to deter customers from using it so they will either 1: be able to use the most recent version or/and 2: Prevent customers from using this version only to realize it's scheduled for deprecation. This is a product that is in the EOL process.

**Comment by gloriliale** [comment]
Hi @mjanssen2021! I've asked our team and the content team about this and will let you know their thoughts asap, thanks for the posting about it!

**Comment by aren-twilio** [comment, marked as answer]
Thanks @mjanssen2021 - great idea. Very happy to add. The main thing will be making sure people stay on top and update labels as the product state changes; for that reason I'd suggest we don't go too granular on states. E.g. will there be benefit to differentiating between beta and private beta? (Genuine question, not meant to be leading)

Also feel free to let me know if you think of any other states. I'll assume we also want one for deprecated products.

**Reply by mjanssen2021** [reply]
Thank you @aren-twilio ! +1 to the thought of folks needing to stay on top of the product labels and not being too granular. I'll completely leave that up to you if thats okay! I'll let you know if I notice any other states that come up. 

Primarily I found with other designers that beta treatments are inconsistent and before [Jo left she gave me this guidance specifically.](https://github.com/twilio-labs/paste/discussions/2142#discussion-3808210) . Appreciate your help!
